JenkinsFile_build_doc.groovy

```diff
@@ -54,7 +57,7 @@ timeout(time: 6, unit: 'HOURS') {
                             scm: [$class: 'GitSCM',
                                 branches: [[name: MERGE_COMMIT]],
                                 userRemoteConfigs: [[refspec: REFSPEC, url: OPENJ9_REPO]]]
```

Comment:
Should this be `OPENJ9_REPO` or `OPENJ9_STAGING`, or does that depend on which build is running?

Reply:
I'm actually concerned about all lines from 48 -> 59 as I'm not sure I understand exactly what is happening. *If launched manually....* what does this mean in the context of the pipeline script? Would we default to master or actually to ghpages?

Reply:
I think the `if` statement is to cover the case where the build is launched manually by a human. This sets a flag to look to master for the latest commit SHA. If the build is launched from a Github webhook, the MERGE_COMMIT variable will be populated automatically from the incoming hook.

l53 is the clone/checkout.

l58 is the code to grab the SHA from master if the flag earlier was set.

So what does it all mean for this new modified build? I think this bit of code is correct at the moment for your needs as I understand it. `MERGE_COMMIT` is used for the build description, the Github status update. and the generated commit message.